librz/core/cconfig.c

```diff
@@ -3747,7 +3762,9 @@ RZ_API int rz_core_config_init(RzCore *core) {
 		NULL);
 	SETPREF("search.prefix", "hit", "Prefix name in search hits label");
 	SETI("search.maxhits", 0, "Maximum number of hits ('0' means no limit)");
-	SETBPREF("search.show_progress", "true", "Show the search process.");
+	n = NODECB("search.show_progress", "1", &cb_search_show_progress);
+	SETDESC(n, "Show the search process (true, false, intervals)");
+	SETOPTIONS(n, "true", "false", "intervals", NULL);
```

Comment:
tbh, i don't like this. is not really clear...

Reply:
how about a12dccf5d976ad7ac852e9ee567ef1250dfe71e7?

Reply:
https://github.com/rizinorg/rizin/commit/a12dccf5d976ad7ac852e9ee567ef1250dfe71e7 is nice IMHO